### DISCUSSION TITLE
ArgoCD-Notifications-Controller replica count

### DISCUSSION THREAD

**Discussion by RiegerSebastian** [argoproj/argo-helm#2080] [General]
Hello Team,
I saw that the replica count of my deployed argoCD-Notification-Controller is 1. After checking the helm chart I see that there is no templating of the replica count for the controller. In the documentation is nothing mentioned and I like to ask what the reason that you did not template the [replica count for the notification-controller](https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-notifications/deployment.yaml#LL16C3-L16C14)? 

As for argocd-dex-server I saw in the documentation that it has an in memory db and can not be deployed with multiple replicas. Does the notification-controller has also a limitation for multiple replicas or can we template the replica count?

Thanks for helping me.
KR
Sebastian

**Comment by pdrastil** [comment]
Because notification controller do not suport HA. If you have more than 1 you might get spammed with duplicate messages. That's the reason why rollout strategy is Replace as other strategy doesn't make sense.
